app/views/users/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2><%= @user.name %></h2>
+
+<% if !@user.created_events.empty? %>
+  <h3>My events:</h3>
+  <ul>
+  <% @user.created_events.each do |event| %>
```

Comment:
Your view is making two calls to `@user.created_events`. You can store the result of that query on an instance variable in the controller. Make your view use as less logic as you can